docs/run_test/runbook.md

```diff
@@ -320,22 +335,107 @@ type: str, optional, default is empty.
 
 Variable name.
 
-### value
+#### value
 
 type: str, optional, default is empty
 
 Value of the paired variable.
 
-### artifact
+### transformer
+
+type: list of Transformer, default is empty
+
+#### type
+
+type: str, required, the type of transformer. See
+[transformers](../../lisa/transformers) for all transformers.
 
 #### name
 
-#### locations
+type: str, optional, default is the `type`.
 
-##### type
+The unique name of transformer. It's used to be depended by other transformers.
+If it's not specified, it will use the `type` field. But if there are two
+transformers with the same name, one of them should have name at least.
 
-##### path
+#### prefix
+
+type: str, optional, default is the `name`.
+
+The prefix of generated variables from this transformer. If it's not specified,
+it will use the `name` field.
+
+#### depend_on
+
+type: list of str, optional, default is None.
+
+The depended transformers. The depended transformers will run before this one.
+
+#### rename
+
+type: Dict[str, str], optional, default is None.
+
+The variables, which need to be renamed. If the variable exists already, its
+value will be overwritten by the transformer. For example, `["to_list_image",
+"image"]` means change the variable name `to_list_image` to `image`. The
+original variable name must exist in the output variables of the transformer.
 
+### combinator
+
+type: str, required.
+
+The type of combinator, for example, `grid` or `batch`.
+
+#### grid combinator
+
+It uses to make full matrix of variables. For example, 2 images and 3 VM sizes
+mean run 2*3=6 times of their combinations.
```

Comment:
```suggestion
```
Since it's not a configurable field, should this explanation be in concepts.md instead of here in runbook?